zerver/lib/export.py

```diff
@@ -1308,13 +1308,16 @@ def export_uploads_and_avatars(realm: Realm, output_dir: Path) -> None:
             output_dir=realm_icons_output_dir,
         )
     else:
+        users = list(UserProfile.objects.filter(realm=realm))
```

Comment:
This doesn't use our standard `select_related`, but maybe that's OK since we only access the IDs in the functions this calls?  I suppose it'd be better to change this interface to just pass a `List[int]` for the user IDs given that observation.

Reply:
fix coming